packages/device/src/components/register.module.css

```diff
@@ -0,0 +1,233 @@
+.container {
```

Comment:
I think we are trying to use scss everywhere instead of css for more modern and flexible css.